app/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -4,26 +4,34 @@
 import com.techcourse.repository.InMemoryUserRepository;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import nextstep.mvc.controller.asis.Controller;
+import nextstep.mvc.view.JspView;
+import nextstep.mvc.view.ModelAndView;
+import nextstep.web.annotation.Controller;
+import nextstep.web.annotation.RequestMapping;
+import nextstep.web.support.RequestMethod;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class LoginController implements Controller {
+@Controller
+public class LoginController {
 
     private static final Logger log = LoggerFactory.getLogger(LoginController.class);
 
-    @Override
-    public String execute(final HttpServletRequest req, final HttpServletResponse res) throws Exception {
+    @RequestMapping(value="/login", method={RequestMethod.POST})
```

Comment:
```suggestion
    @RequestMapping(value = "/login", method = RequestMethod.POST)
```
줄정렬이 적용 안된 것 같네요!
개인 취향이신건지 아닌지 긴가민가해서 코멘트 덧붙이자면 메서드가 하나인 경우면 굳이 { }를 붙이지 않으셔도 됩니다😄

Reply:
다른 부분은 다 뺐는데 왜 붙여있었지.. 제거했습니다~